packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -280,6 +350,65 @@ export class NotifiFrontendClient {
     await Promise.all([saveAuthorizationPromise, saveRolesPromise]);
   }
 
+  async fetchData(): Promise<FrontendClientData> {
+    const [
+      rawAlerts,
+      rawConnectedWallets,
+      rawSources,
+      rawTargetGroups,
+      rawEmailTargets,
+      rawSmsTargets,
+      rawTelegramTargets,
+      rawDiscordTargets,
+      rawFilters,
+    ] = await Promise.all([
+      this._service.getAlerts({}),
+      this._service.getConnectedWallets({}),
+      this._service.getSources({}),
+      this._service.getTargetGroups({}),
+      this._service.getEmailTargets({}),
+      this._service.getSmsTargets({}),
+      this._service.getTelegramTargets({}),
+      this._service.getDiscordTargets({}),
+      this._service.getFilters({}),
```

Comment:
ah, this pattern was due to an old server-side bug which prevented us from querying multiple fields at once.

Insetad, let's define a `service.fetchData` which makes a composite query

i.e. something like:
```
query FetchData {
  alerts { ...AlertFragment }
  connectedWallets { ... }
  sources { .... }
  ...
}

Reply:
Addressed!